packages/terra-form-input/CHANGELOG.md

```diff
@@ -3,6 +3,8 @@ ChangeLog
 
 Unreleased
 ----------
+### Added
+* maxWidth prop added to input field component
```

Comment:
Should this change be applied to `terra-form-field` so all fields have the ability to have a maxWidth. As is this functionality will only work for `terra-form-input` and no other Field components. (SelectField, RadioField, TextArea)

Reply:
Logic moved to terra-form-field!